include/mgp.hpp

```diff
@@ -2486,6 +2528,21 @@ inline bool Map::operator==(const Map &other) const { return util::MapsEqual(ptr
 
 inline bool Map::operator!=(const Map &other) const { return !(*this == other); }
 
+inline const std::string Map::ToString() const {
+  std::string return_string = "{";
+  const size_t map_size = Size();
+  size_t i = 0;
+  for (auto item : *this) {
```

Comment:
Can you change this with while loop same as in List?

Reply:
Isnt it better to keep the for lopp, cause I need to iterate through to the semi last element for string formatting, and with iterators I would have to do additional checks if the element is semi last